bigquery_etl/cli/query.py

```diff
@@ -506,76 +507,89 @@ def _backfill_query(
     run_checks,
 ):
     """Run a query backfill for a specific date."""
+    backfill_date_str = backfill_date.strftime("%Y-%m-%d")
+    if backfill_date_str in exclude:
+        click.echo(f"Skip {query_file_path} backfill for run date {backfill_date_str}")
```

Comment:
```suggestion
        click.echo(f"Skipping {query_file_path} backfill for run date {backfill_date_str}")
```

Reply:
There's something off here. Like this function should be simple: run a backfill on a date. The filtering of dates should happen outside of this function, e.g.:
```
dates.filter(date -> date not in excludes).map(date -> _backfill_query_closure(date))
```

Reply:
Would also let us remove the early return 😉  (but not blocking!)

Reply:
Dates could be filtered earlier here yeah, I assume the original motivation was to allow logging which I think is still useful. I think in `_backfill_query`, `exclude` could probably be changed to a boolean. Something like `skip=False`